packages/iov-keycontrol/src/hdpaths.ts

```diff
@@ -54,7 +66,26 @@ export class HdPaths {
    * compatible to the 5 component BIP44 standard.
    */
   public static bip44Like(coinType: number, account: number): readonly Slip10RawIndex[] {
-    return [Slip10RawIndex.hardened(44), Slip10RawIndex.hardened(coinType), Slip10RawIndex.hardened(account)];
+    return [
+      Slip10RawIndex.hardened(HdPaths.purposes.bip44),
+      Slip10RawIndex.hardened(coinType),
+      Slip10RawIndex.hardened(account),
+    ];
+  }
+
+  /**
+   * This function is the same as bip44Like but allows custom purposes, e.g. for use by the faucet
+   */
+  public static bip44LikeWithCustomPurpose(
```

Comment:
_BIP44 with custom purpose_ does not make much sense, since no aspect of BIP44 is left then. It is a plain BIP43.

From BIP44:

> This BIP is a particular application of BIP43. 

However, I don't see how this function is useful at all, since the faucet needs a 4-component path anyways.